addons/l10n_nl/demo/demo_company.xml

```diff
@@ -2,11 +2,11 @@
 <odoo>
     <record id="partner_demo_company_nl" model="res.partner">
         <field name="name">NL Company</field>
-        <field name="vat">NL43603B11</field>
+        <field name="vat">NL219987701B73</field>
```

Comment:
This number isn't vies valid either.

Reply:
Think we can not use a real VAT number. A real number is from a real company. Any ideas?